closed/GensrcJ9JCL.gmk

```diff
@@ -57,7 +57,16 @@ $(eval $(call SetupCopyFiles,COPY_OVERLAY_FILES, \
 		src/java.base/share/classes/sun/security/provider/DigestBase.java \
 		src/java.base/share/classes/sun/security/provider/SecureRandom.java \
 		src/java.base/unix/classes/java/lang/ProcessEnvironment.java \
+<<<<<<< Upstream, based on openj9
 		src/java.base/unix/classes/sun/security/provider/NativePRNG.java \
+=======
+		src/java.se/share/data/jdwp/jdwp.spec \
```

Comment:
Please squash so we don't have commits with unresolved merge issues like this.

Reply:
Squashed to remove unresolved merge conflict.